api-first-approach/index.asciidoc

```diff
@@ -0,0 +1,90 @@
+= API first approach with OpenAPI generator
+====
+This tutorial will show you how to use an API first approach to create a RESTful web service in Quarkus from an OpenAPI specification using the OpenAPI generator plugin.
+
+For more information about the API fist approach, see the [devon4j documentation](https://github.com/devonfw/devon4j/blob/master/documentation/guide-api-first.asciidoc).
+For more details on the OpenAPI generator, have a look at the [official webpage](https://openapi-generator.tech/).
+
+## Prerequisites
+* Installed devonfw-ide (or at least Java and Maven installed)
+
+## Learning goals
+* You will learn how to generate REST services from OpenAPI specifications using OpenAPI generator
+
+====
+
+[step]
+--
+restoreDevonfwIde(["java", "mvn"])
+--
+
+====
+In the first step, create the initial Quarkus project using the quarkus-maven-plugin on the command line.
+
+When using this plugin, you need to pass the groupId and the artifactId. We additionally add the Quarkus resteasy-jackson extension, because we need it later.
+
+[step]
+== Create the Quarkus project
+--
+executeCommand("mvn io.quarkus.platform:quarkus-maven-plugin:2.6.1.Final:create \"-DprojectGroupId=com.devonfw.quarkus\" \"-DprojectArtifactId=api-first-tutorial\" \"-Dextensions=resteasy-jackson\"","mvn io.quarkus.platform:quarkus-maven-plugin:2.6.1.Final:create -DprojectGroupId=com.devonfw.quarkus -DprojectArtifactId=api-first-tutorial -Dextensions=resteasy-jackson")
```

Comment:
Shouldn't we use the devon command to create the project?

Reply:
The devon command can only create Spring projects at the moment. That's why I did it this way.